core/src/main/java/com/graphhopper/routing/weighting/GenericWeighting.java

```diff
@@ -124,40 +119,16 @@ public double calcWeight(EdgeIteratorState edgeState, boolean reverse, int prevO
         if (time == Long.MAX_VALUE)
             return Double.POSITIVE_INFINITY;
 
-        int spatialId = gEncoder.getSpatialId(edgeState.getFlags());
```

Comment:
Please for now without GenericWeighting. So like I linked: determine the country and apply a speed reduction if necessary (similar with the access).